### PR TITLE
Remove cloudformation from update ami scripts

### PR DESCRIPTION
**Pull request by stevenGravy**
Will include in backports for removing cloudformation after put into `master`

### CODE REVIEW

File: assets/aws/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# VPC ID used for builds
+ VPC ID used for builds
```
Comment (zmb3) [inline assets/aws/Makefile:1]: Mistake?

File: assets/aws/update-ami-ids.sh [hunk 2]
```diff
@@ -8,16 +8,12 @@ set -euo pipefail
 # -- (something like "export PATH=/usr/local/opt/findutils/libexec/gnubin:$PATH")
 
 # shellcheck disable=SC2086
-usage() { echo "Usage: $(basename $0) [-a <AWS account ID>] [-m <cloudformation/terraform>] [-t <oss/ent/ent-fips>] [-r <comma-separated regions>] [-v version]" 1>&2; exit 1; }
+usage() { echo "Usage: $(basename $0) [-a <AWS account ID>] [-t <oss/ent/ent-fips>] [-r <comma-separated regions>] [-v version]" 1>&2; exit 1; }
 while getopts ":a:m:t:r:v:" o; do
```
Comment (zmb3) [inline assets/aws/update-ami-ids.sh:12]: Remove the m

**Review by zmb3 — CHANGES_REQUESTED** [review]

File: assets/aws/Makefile (hunk 1 above)
Comment (stevenGravy) [reply assets/aws/Makefile]: Yes, fixed, thanks!

File: assets/aws/update-ami-ids.sh (hunk 2 above)
Comment (stevenGravy) [reply assets/aws/update-ami-ids.sh]: removed

**Review by r0mant — APPROVED** [review]
I ran `TELEPORT_VERSION=11.2.2 make -C assets/aws create-update-pr` off of this PR's branch and it worked fine: https://github.com/gravitational/teleport/pull/20466. Although it included the changes from here as well so we can probably just merge it as a part of my PR @stevenGravy.

**Comment by stevenGravy** [comment]
> I ran `TELEPORT_VERSION=11.2.2 make -C assets/aws create-update-pr` off of this PR's branch and it worked fine: https://github.com/gravitational/teleport/pull/20466. Although it included the changes from here as well so we can probably just merge it as a part of my PR @stevenGravy.

Sounds great! Closed this one

**Pull request closed without merging**